README.md

```diff
@@ -161,18 +161,37 @@ The following tasks are available:
 
 ## Testing
 
-To perform manual testing of the latest version of each component:
+Visual regression testing is implemented by [BackstopJS](https://github.com/garris/BackstopJS). In order to avoid browser rendering inconsistent issue on different environments, the test needs to run inside of docker container.
 
-```
-gulp dev
-```
+### Setup docker
 
-To perform manual testing of the backwards-compatible bundle package:
+Install docker on your machine [Download Docker](https://hub.docker.com/search/?type=edition&offering=community&architecture=amd64)
 
-```
-cd bundles/spectrum-css-compat
-gulp dev
-```
+### Backstop reference bitmap
+
+The reference bitmaps are hosted in a adobe internal repo and published as [@spectrum-css/spectrum-css-vr-test-asset](https://www.npmjs.com/package/@spectrum-css/spectrum-css-vr-test-asset). All the reference bitmaps are generated within docker instance. If you want to run test with native environment, you have to regenerate the reference yourself.
+
+### Start dev server with Browsersync notification turnoff
+
+`BROWSERSYNC_NOTIFY=false gulp dev`
```

Comment:
`BROWSERSYNC_NOTIFY` no longer required, right?

Reply:
> This looks good! Just some docs updates, a stray `console.log`, and some nits.
> 
> Is there anything else you need to do before we can merge this, @jianliao? Should we merge the `yarn` stuff over in #279 first?

Yes, I think #279 should be merged first.